### ISSUE TITLE
PIXI.Graphics() .arc() is a closed shape now (regression, was OK in 3.0.6)

### ISSUE THREAD

**Issue by Udo** [pixijs/pixi.js#1985]
```javascript
var graphics = new PIXI.Graphics();
graphics.lineStyle(2, 0xFF00FF, 1);
graphics.arc(200, 200, 100, 0, Math.PI);
```
^ this should draw a bowl shape that's open at the top, which works in 3.0.6. In 3.0.7, however, the two endpoints are joined with a straight line and I don't see any option to work around that.

**Comment by englercj** [comment]
This comes from #1863.

**Comment by trakout** [comment]
@Udo @englercj Ran into this today on 3.0.10. 

I couldn't replicate with @Udo's example (calling arc() directly after initiating a new Graphics() object), but was able to replicate by storing Graphics() objects in an array to animate later on. Within the animation loop, I had the following:

```javascript
var r = 20;
graphics.clear();
graphics.lineStyle(6, 0xffffff, 1);
graphics.arc(0, 0, 20, 0, Math.PI);
```

This would without fail close the arc. I checked & verified that `graphics.currentPath.shape.closed = false`.

After looking at [this](https://github.com/pixijs/pixi.js/commit/cee82b8612e55a6a533ca4c49a69ff632897c722#diff-7985dd8419219c20229060928bac2bf0R10179), I had a hunch that line 10179 was not running. Still not sure if it is, but adding moveTo(radius, 0) fixed the issue.

So my final code was:
```javascript
var r = 20;
graphics.clear();
graphics.lineStyle(6, 0xffffff, 1);
shapeRef.moveTo(20, 0);
graphics.arc(0, 0, 20, 0, Math.PI);
```

I'm still fairly confused as to why starting an arc acts like it has no initial points in the case. If it doesn't, what's the point of setting arc(cx, cy, ...)?

One last note, I tried without moveTo() and used 200 for cx/cy (basically any value aside from 0), same issue.
